src/arrayMethodSort.js

```diff
@@ -6,6 +6,34 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
     // write code here
+
+    if (!compareFunction) {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = 0; j < this.length - i - 1; j++) {
+          if (String(this[j]) > (String(this[j + 1]))) {
+            const temp = this[j];
+
+            this[j] = this[j + 1];
+            this[j + 1] = temp;
+          }
+        }
```

Comment:
Don't duplicate this part of code, you can create a separate function and set it as default value for compareFunction